rkt/run.go

```diff
@@ -130,7 +131,7 @@ func runRun(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
-	s, err := store.NewStore(globalFlags.Dir)
+	s, err := store.NewStore(tempDir)
```

Comment:
I think `tempDir` is `""` not defined if `getDataDir() != ""`. Before, `store.NewStore` was never called with an empty string.